### PR TITLE
explore tab navigation section

### PR DESCRIPTION
**Pull request by JMurphyWeb**
#667 

Update styles for the tab navigation (eg in in `/orgs` route)

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -277,21 +277,23 @@ Light-teal:     #66d9e0;
     Toggle Bar in browse view
   |||||||||||||||||||||||||||||||||||||||||||*/
   .togglebar {
-    background: #eaeaea;
+    background: #f8f8f8;
```
Comment (mk4111) [inline public/css/main.css]: I think this colour might be new? Can we add this to the colour list at the top of main.css?

File: public/css/main.css [hunk 2]
```diff
@@ -277,21 +277,23 @@ Light-teal:     #66d9e0;
...
   }
   .togglebar__item--selected, .togglebar__item--selected:hover {
-    background: #fafafa;
+    background: #ffffff;
   }
   .togglebar__link, .togglebar__link:visited  {
     padding: 1.2rem 0;
-    color: #6F8B9D;
+    color: #40667e;
+    opacity: 0.5;
```
Comment (harrygfox) [inline public/css/main.css]: From stackoverflow (http://stackoverflow.com/questions/13385492/css-opacity-vs-rgba-which-one-is-better):
"Narrowing down the area of effect [by only changing an element's colour, not a whole element's opacity] is a warranty of performance boost ... and makes it much easier for browsers to figure out when they need to get redrawn...  which I've confirmed by trying both options and noticing that website using rgba() in place of opacity felt noticably smoother, especially on Safari and Opera."

So, if this (or any others like it) needs to be partly transparent: use `rgba` over `opacity`.

**Moreover,** i'd recommend using `#9BAEBA` which is the true colour of the half-opaque link on an `f8f8f8` background. In this way the colour of this link is not _half_ determined by it's background colour.

File: templates/partials/browse-tab-navigation.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!-- Change filter tag link (dropdown) -->
 <div class="card card--header daintree--blue">
```
Comment (harrygfox) [inline templates/partials/browse-tab-navigation.html]: **\*BEM Error\*:** modifier class with no block / element

Needs a `daintree`
...what's a daintree?

**Comment by JMurphyWeb** [comment]
@harrygfox there is going to be a change to the title class in one of @mk4111's PRs so I will revisit the height of the tags title at the top after that has been merged.

**Comment by harrygfox** [comment]
@JMurphyWeb Merged innit

**Pull request merged**